chi_analysis/calculate_shapelimit_qbh.py

```diff
@@ -0,0 +1,115 @@
+import os, sys
```

Comment:
This script can actually be dropped. The calculate_shapelimit_13TeV script can also calculate the QBH in one go.